app/domain/errors.rb

```diff
@@ -383,6 +383,17 @@ module AuthnJwt
         code: "CONJ00072E"
       )
 
+      NoSuchFieldInToken = ::Util::TrackableErrorClass.new(
+        msg: "No such field '{0}' in the token",
+        code: "CONJ00073E"
+      )
+
+      NoRelevantIdentityProvider = ::Util::TrackableErrorClass.new(
+        msg: "No relevant identity provider found." \
+              "Please add `token-app-property` variable and populate it" \
+              "with secret containing field name in jwt decoded token",
+        code: "CONJ00074E"
```

Comment:
@shulifink please review these 2 msgs

@tzheleznyak 
I'm not sure I understand this sentence.
what does "secret containing field name" means?

Reply:
I will remove the word secret. But we using the same mechanisem that saves secret for this field name so thats why i wrote secret .